libexec/anyenv-install

```diff
@@ -113,7 +113,7 @@ definitions() {
 
 echoe() {
   local message="$1"
-  (>&2 echo -e "\e[33m${message}\e[0m")
+  (>&2 printf "\e[33m${message}\e[0m")
```

Comment:
If `message` includes `%`, it causes problem.
So I think `printf "\e[33m%s\e[0m" "${message}"` is better.

Reply:
Thank you for your suggestion.
I changed to use format specifier.

Also I added `¥n` because `printf` doesn't create a newline unlike `echo`.